### DISCUSSION TITLE
How to support both a select field and custom entry field in a array model field

### DISCUSSION THREAD

**Discussion by lokeshwarobuli** [bhch/django-jsonform#169] [Q&A] [answered]
Based on comment from https://github.com/bhch/django-jsonform/issues/72. I am overriding the schema like the attached screenshot. Is there a way where I can allow the user to enter their own custom option, incase they couldn't find what they want from the choices as well as allow to select from choices ? Right now I am able to achieve only either one of them by adding items key
<img width="684" alt="Screenshot 2024-05-31 at 4 38 20 PM" src="https://github.com/bhch/django-jsonform/assets/85650351/68f820b2-e350-473c-bdf2-e5d7b740fcf6">

**Comment by bhch** [comment, marked as answer]
Currently, allowing users to add values other than specified in the choices is not possible.

I think we can make use of `oneOf` keyword for this purpose. 

I'll add this feature in the next version.

**Reply by lokeshwarobuli** [reply]
Thanks @bhch , that would be a very helpful
